### ISSUE TITLE
landfill under  offshore pump

### ISSUE THREAD

**Issue by peternlewis** [christoph-frick/factorio-blueprint-tools#22]
If you add landfill under entities, it includes under the offshore pump, which means the pump cannot be placed. Eg:

0eJyVj9uqwjAQAP9ln6PQ1qY1vyIiVbfHhXQTmlQsJf9uWi8IR8W+ZUNmMjvAXndoW2IPagA6GHagNgM4+uNKj3e+twgKyGMDArhqxsnUtTuZFhe2aywEAcRHvIBKwlYAsidPeBNNQ7/jrtljGx98UAiwxkXK8PhnNC2klMtcQB+P6zJd5mFUTxXqJVrAGVs3YWmZrIp1WmRZIrM8FeBJ3yP+u8uHeRWeRbriY01ax33eIcV8RM5HnmHZ/LDfEfkV2YZwBTMbqSM=

**Comment by christoph-frick** [comment]
I'll look into this, but maybe just better docs are needed.  Have you tried with "sparse"? See https://github.com/christoph-frick/factorio-blueprint-tools/issues/16#issuecomment-714496803

**Comment by christoph-frick** [comment]
I have added better help text to the options.  If you see need to support "full" filling mode but keep pump gaps, please let me know.

**Comment by xiaoshui05** [comment]
landfill not work well when offshore pump outlet is up or down.
```
0eNqd2d1u2kAQQOF32Wun2l/PmleJqooE01oyxjKmKkJ+99rQokhhsxzuQuQ5cfQNLIizemuPdT803ahWZ9W877uDWr2e1aH52a3b5Xfjqa/VSjVjvVOF6ta75dF+uz382g/1S3/c9WoqVNNt6j9qZabvhaq7sRmb+hq6PDj96I67t3qYL7gl+mbuFqrfH+aL993yp+bAi4n+WyjUab5U5p+mqfjUsPlGyDVcvmFzDZ9vuFwj5Bs61yjzDZNrSLYhMdeI+UaVa1T5RplrGE0iIRHJb6pI9k5s4hnz9c7O3fkptWmG+v16ib/XdqDtYNuDtoHt8Hj7w7481i5BW2BbyFZJYiEiiZSJSPXMVi2vI3dfTfUze5SsmWc2J1mzz+xKsuae2Y5kzQPK5V+9GwkkkrqTEhyqC+HdiIBTNRmJ4FhNRipwrqYiToODNRkx4GRNRiw4WpMRB87WZARtbCqCNtYmIiU4XK93Mr+vHJv235vKzwzl/4N4upXbdbfZNm2rlhtITwQ8UeIJoRNR4wmDJyyecA9PBOwRsEfAHgF7BOwRsEfAHgF7eOzhsYfHHh57eOzhsYfHHh57OOzhsIfDHg57OOzhsIfDHg57WOxhsYfFHhZ7WOxhsYfFHhZ7GOxhsIfBHgZ7GOxhsIfBHgZ7aOyhsYfGHhp7aOyhsYfGHpp6SEU9bhMBT5R4QujE4x63CYMnLJ543CNij4g9IvaI2CNij4g9IvaI2EOwh2APwR6CPQR7CPYQ7CFfecyf1C9fFq0+fLdUqN/1cLg0bDReKisuel16PU1/ASi/NQk=
```
![factorio_XJj1LaZvrm](https://user-images.githubusercontent.com/11335012/118383122-cdf37480-b62d-11eb-832d-c922b36d4a2d.png)
![factorio_psDgfrF7GJ](https://user-images.githubusercontent.com/11335012/118383124-d21f9200-b62d-11eb-8974-9fcdc905df58.png)
![factorio_0NxQ6JsJ70](https://user-images.githubusercontent.com/11335012/118383128-d481ec00-b62d-11eb-857c-652bc249d075.png)

**Comment by christoph-frick** [comment]
@xiaoshui05 The problem got fixed in #24
